### DISCUSSION TITLE
GUI front end for data entry?

### DISCUSSION THREAD

**Discussion by ecbettinger** [sqlitecloud/discussions#41] [Help]
I'm new to database administration, and looking for some advice, as well as better understanding the problem I have.  

I have a small research project (will be maybe a total of 800 rows when finished, about 10 tables). I've built a SQLite database and begun to enter data. The data entry process is a bit complex, and requires human interpretation and manual input for each entry. I have several team members in different physical places to help complete the data entry. 

I am currently hosting the DB in SQLite Cloud. I would like to create a form that data enter-ers can use to input data, and have that one form automatically populate multiple tables in the DB. 

Is this possible without code (or with very little code)? What would be the best way to do it? I tried to connect a Wordpress form (Forminator) to SQLite Cloud through Webhooks, but couldn't figure out how to hook it up in such a way that it would be able to populate multiple tables. If each user sets up LibreOffice Base on their machine, would they be able set up a form that could populate the db in SQLite Cloud? 

In advance, thanks for any tips, or workflows to explore.

**Comment by jacobprall** [comment]
Apologies for the delay, @ecbettinger, so you are looking for a solution to streamline data entry into SQLite Cloud for a handful of data entry specialists?
